src/lib/services/project-service.ts

```diff
@@ -508,6 +500,11 @@ export default class ProjectService {
             userId,
         );
 
+        const ownerRole = await this.accessService.getRoleByName(
+            RoleName.OWNER,
```

Comment:
this seems to be a bug since we're getting a role for the owner not the user we're deleting